apps/daimo-mobile/src/view/screen/onboarding/IntroPages.tsx

```diff
@@ -62,14 +84,26 @@ export function IntroPages({
           <View style={styles.introButtonsWrap}>
             <ButtonBig
               type="primary"
-              title="Create Account"
+              title="ACCEPT INVITE"
```

Comment:
was this really the gap you think?
my observations:
- a number of people (at least 3) clicked "open in app" on the web page -- we used to say "Already have it? Open in app", I think that copy was better to prevent that
- on the invite page itself people would click into the text bar and not notice the button below, I think we should just delete the text input or make it one clicks away, a secondary / text button with copy like "Manually type invite?"
- On create account we should add a pending Op to prevent users from getting confused why they didn't get their signup bonus

Reply:
see designs in #615

Reply:
first and third point sgtm, will do on this PR